src/components/field-types/RadioFieldComponent.vue

```diff
@@ -79,6 +79,11 @@
     watch: {
       localValue () {
         this.$emit('input', this.localValue)
+
+        this.fieldState.$dirty = true
```

Comment:
why is this needed ? the vue-form should do this , if it does not please add a not why this work around is needed.

Reply:
I understand why $touched does not become true. It triggers on blur and if you click a radio button it never becomes focused. The $dirty should become true when the value changes and I think this is probably a bug in the vue form library.

Reply:
Added note